site/app/templates/polls/ViewPoll.twig

```diff
@@ -97,6 +102,25 @@
                     <button type="submit" class="btn btn-primary student-submit">Submit</button>
                 {% endif %}
             </form>
+            {% if not user_admin and poll.isOpen() and poll.getAllowsCustomResponses() %}
+                <div class = 'info-wrapper custom-response-wrapper'>
+                        <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
+                        <input type="hidden" name="poll_id" value="{{ poll.getId() }}"/>
+                        <br />
+                        <div class="response-container">
+                            <textarea aria-label="Response text" class="poll-response custom-poll-response" name="custom-response" placeholder="Enter response here..." rows="10" cols="30"></textarea>
+                        </div>
+                        <i>
+                            Are you certain you want to add a response? It will be visible to every student.<br />
+                        </i>
+                        <button type="button" class="btn btn-primary student-submit custom-response-submit" onclick = "addCustomResponse({{poll.getId()}},'{{base_url}}');">Add Response</button>
+                    <script>
+                        document.addEventListener('DOMContentLoaded', function() {
+                            validateCustomResponse();
+                        });
```

Comment:
Any reason why `$(document).ready()` shouldn't be used instead here?

Reply:
I don't think there was a particular reason. Changed in [e5d53f8](https://github.com/Submitty/Submitty/pull/10197/commits/e5d53f84f79e906d924e9b85edc2927df431bf4c)